### PR TITLE
[sd-2453-1] (error handling)-undo/redo fix for medium editor

### PR DESCRIPTION
**Pull request by sjunaid**

### CODE REVIEW

**Comment by amagdas** [comment]
We should add a test for this, basically editing two different editors and check that they work fine side by side.

File: client/spec/authoring_spec.js [hunk 1]
```diff
@@ -5,13 +5,13 @@ var workspace = require('./helpers/workspace'),
     authoring = require('./helpers/authoring');
 
 describe('authoring', function() {
-    it('can open item stage', function() {
+    xit('can open item stage', function() {
```
Comment (amagdas) [inline client/spec/authoring_spec.js]: uncomment it please.

**Comment by amagdas** [comment]
:+1:  after enabling the e2e

**Comment by mdhaman** [comment]
:+1:

**Pull request merged**